CONTRIBUTING.md

```diff
@@ -1,48 +1,68 @@
 # Contributing to Elastic Common Schema (ECS)
 
-All information related to ECS is versioned in this repository. All changes to ECS
-happen through Git and Pull Requests.
+All information related to ECS is versioned in the [elastic/ecs](https://github.com/elastic/ecs) repository. All changes to ECS
+happen through Pull Requests submitted through Git.
 
 
 ## Requirements
 
-To contribute to ECS the following tools are expected to be running on your machine:
+You need these tools to contribute to ECS:
 
 * [Git](https://git-scm.com/)
 * [Python](https://www.python.org/)
 * [Go](https://golang.org/)
 
 ## Steps to contribute
```

Comment:
Should we have a main title `Contribute` and below the two ways. Not sure yet how we should call the two. Ideas?

Reply:
I was thinking of "Steps to contribute" as the *preferred* (main) path for people who are capable of contributing through a PR, with "Other ways to contribute" as an alternate (if...else) path. That's why I formatted "Other ways" as a subheading to "Steps...".
If you feel strongly that contributing through a PR and contributing through an issue should be at the same level heading, I will make that happen.

Reply:
I'm on the fence about this one. For code parts I like PR's directly as they make discussing easier. But this is not really code and often an issue is the right part to get started as it's more question based initially. 

But in the end I don't think it will make a real difference on what we call it here. So let's move forward with the way it is.

Reply:
Thanks for taking time to explain.  Your insights help me understand the project better, and will help me add more value going forward.